distribution/std-lib/Base/src/Time/Time.enso

```diff
@@ -0,0 +1,177 @@
+from Base import all
+import Base.Time.Date
+import Base.Time.Duration
+import Base.Time.Time_Of_Day
+import Base.Time.Zone
+
+polyglot java import java.time.ZonedDateTime
+polyglot java import java.time.format.DateTimeFormatter
+polyglot java import org.enso.base.Time_Utils
+
+type Time
+
+    ## A date-time with a time-zone in the ISO-8601 calendar system, such as
+       "2007-12-03T10:15:30+01:00 Europe/Paris".
+
+       Time is a representation of a date-time with a time-zone. This class
+       stores all date and time fields, to a precision of nanoseconds, and a
+       time-zone, with a zone offset used to handle ambiguous local
+       date-times.
+
+       For example, the value "2nd October 2007 at 13:45.30.123456789 +02:00 in
+       the Europe/Paris time-zone" can be stored as Time.
+    type Time internal_zoned_date_time
+
+    ## Get the year field.
+    get_year : Integer
```

Comment:
Same as before. Get rid of `get` in getters and expand all of `nano` and `milli` to full names.